sdk/storage/AzureStorageBlob/Source/Data/Transfer.swift

```diff
@@ -30,11 +30,14 @@ import Foundation
 // MARK: Protocols
 
 public protocol Transfer: AnyObject {
-    var rawState: Int16 { get set }
+    var id: UUID { get }
```

Comment:
It does. I thought about giving it another name, but then I realized that swiftlint is making the wrong decision for the consumer. As a consumer of any public API, I would very much expect that if an object had an "identifier" property, it would be called `id`.

Reply:
Yeah, I agree.